src/services/payment-flow/schema.ts

```diff
@@ -1,4 +1,4 @@
-import * as mongoose from "mongoose"
+import mongoose from "mongoose"
```

Comment:
shoudn't this file be in the mongoose service?

Reply:
I guess this was intentional, probably a long term preparation for migration to PostgreSQL. This service should be independent but maybe @vindard or @UncleSamtoshi could confirm (I was not involved in that development) 

ps: if a change is required here probably should be done in an additional PR

Reply:
We have `schema.ts` files that import from `mongoose` in the following packages:
- `src/services/lnd`
- `src/services/ledger`
- `src/services/payment-flow`

And then we have other schemas in the `mongoose` package like:
- `WalletSchema`
- `WalletInvoiceSchema`
- `UserSchema`

We should probably standardize on where we define these schemas.